projectsmember/Project Manager.aspx.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace projectsmember
+{
+    public partial class billing : System.Web.UI.Page
+    {
+        SqlConnection LOGConn = new SqlConnection(ConfigurationManager.ConnectionStrings["sojaIrishConnectionString"].ConnectionString);
+
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            if (Session["adminstatus"] != null &&
+               Session["adminstatus"].ToString() == "login_admin")
+            {
+                //successful login
+            }
+            else
+            {
+                Response.Redirect("login_admin.aspx");
+            }
+
+
+            if (LOGConn.State == ConnectionState.Open)
+            {
+                LOGConn.Close();
+            }
+            LOGConn.Open();
+
+            DataSet1TableAdapters.ProjectsTableAdapter dstaProjNotif2 = new DataSet1TableAdapters.ProjectsTableAdapter();
```

Comment:
Ok, so here, we don't need LOGConn at all. 

we should be able to do those all with dataset object. You need to define a new query there, where you pass username and password and the rest is taken care of.  Just like how I did it in this Video: 

https://www.youtube.com/watch?v=0NnwpeqEFOs&ab_channel=SOJA

Reply:
Was corrected